picasso/src/test/java/com/squareup/picasso/RequestCreatorTest.java

```diff
@@ -544,7 +554,7 @@ public void intoRemoteViewsNotificationWithFitThrows() {
   }
 
   @Test
-  public void intoTargetNoResizeWithCenterInsideOrCenterCropThrows() {
+  public void intoTargetNoResizeWithCenterInsideOrCenterCropOrOnlyScaleDownThrows() {
```

Comment:
This should probably also be named `intoTargetNoResizeOrFitWithCenterInsideOrCenterCropOrOnlyScaleDownThrows`.